plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -60,7 +60,7 @@ class ScaleBarImpl : ScaleBar, View {
     }
 
   override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
-    val width = mapViewWidth / 2
+    val width = mapViewWidth * settings.ratio
```

Comment:
should we add tests?

Reply:
It's hard to test this part before we have a pixel comparison test.

Reply:
why it's hard to test? mock `settings` and verify that called `setMeasuredDimension` with some values

Reply:
we can extract the logic to a method and test that method instead.